deploy/postgres/postgres-deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
                   name: assisted-installer-rds
                   key: db.password
           volumeMounts:
-            - mountPath: /var/lib/postgress
+            - mountPath: /var/lib/pgsql/data
```

Comment:
out of curiosity, is this change required? Wondering what it's related to.

Reply:
/var/lib/postgress looked fishy to me. From the [docs] (https://hub.docker.com/_/postgres), the default should have been /var/lib/postgresql/data. When I actually looked at the postgres container logs, it was using /var/lib/pgsql/data. Looking at the docs more closely, I think we should also be setting a value for PGDATA.

Reply:
> Looking at the docs more closely, I think we should also be setting a value for PGDATA.

I think it's safer to set it, yes. I don't think this value will be changed upstream, which would break the current configs.


On that note, I wonder if we want to make this path configurable, instinct says we better don't do this yet unless there's a real need for it in the future.

Reply:
One of the QE folks is having problems deploying postgres using a NFS backed PV:

`11:39:11.715 UTC [72] FATAL:  data directory "/var/lib/pgsql/data/userdata" has wrong ownership`

Seems to point to setting PGDATA. Will need further investigation.